src/sql/azdata.proposed.d.ts

```diff
@@ -462,6 +486,13 @@ declare module 'azdata' {
 			 */
 			None = 'None'
 		}
+
+		/**
+		 * Opens the change password dialog in connection management service.
+		 * @param initialConnectionProfile The connection profile to change the password for and connect to.
+		 * @param options The connection options containing the required connection parameters.
+		 */
+		export function openChangePasswordDialog(initialConnectionProfile: IConnectionProfile, options: IConnectionCompletionOptions): void;
```

Comment:
In order to connect using the password dialog with the current design of the connection dialog, you are required to pass in parameters. Since Alan asks that we need to preserve them from the connection management service. They will have to be passed in as well unfortunately.